index.bs

```diff
@@ -2220,7 +2226,13 @@ Base64-encoded string.
 
       browsingContext.CaptureScreenshotParameters = {
         context: browsingContext.BrowsingContext,
-        ? clip: browsingContext.ClipRectangle
+        ? format: "png" / "webp" / browsingContext.JpegFormat .default "png",
```

Comment:
I think if we're going to have JPEG use an object it's going to be easier and more consistent to use an object in all cases, even when we don't (currently) have adjustable parameters for the other formats.

Reply:
This example follows from `origin` in PerformActions which has `"viewport" / ElementOrigin`

Reply:
Yeah, although that's for compatibility with classic. Anyway I guess this is probably OK and we can extend it in the future if required.